### PR TITLE
Less snark

### PR DESCRIPTION
**Pull request by mccalluc**
@caseyedavis12 : here's what it looks like:
screen shot 2016-03-04 at 1 56 58 pm

@afred: perhaps hold off until we hear back? Though I want to make another PR to make these metadata only until then.

### CODE REVIEW

**Comment by mccalluc** [comment]
(This doesn't store the confirmation in a cookie: it should, eventually.)

**Comment by caseyedavis12** [comment]
@mccalluc: Looks great to me. Passed this along to legal.

File: app/views/catalog/_password_modal.html.erb [hunk 1]
```diff
@@ -5,17 +5,29 @@
...
       <div class="modal-footer">
-        <input id="password" type="password"></input>
+        I have read and agree to the terms and conditions:
+        <input id="read-and-agree" type="checkbox"></input>
+        <input id="password" type="password" placeholder="Password from above"></input>
         <button type="submit" class="btn btn-primary" onclick="
-          if ($('#password').val() === $('#hint').text()) {
+          // When I copy-and-paste, I get a space character at the beginging...
+          // not sure why, but replace cleans it up.
+          if ($('#password').val().replace(/\W/g,'') === $('#hint').text() &amp;&amp; $('#read-and-agree').is(':checked')) {
```
Comment (afred) [inline app/views/catalog/_password_modal.html.erb]: @mccalluc the escaped `&` confuses me. For this to work, the `&amp;` will have to get unescaped, right? When does that happen?

**Comment by afred** [comment]
@caseyedavis12 @mccalluc Just want to all be on the same page that the password usage here is not a common pattern, and provides no additional protection of the content.

File: app/views/catalog/_password_modal.html.erb (hunk 1 above)
Comment (mccalluc) [inline app/views/catalog/_password_modal.html.erb]: Remember this whole thing is an XML document... so the javascript sees "&&".

Comment (afred) [inline app/views/catalog/_password_modal.html.erb]: @mccalluc ah.. right. Curious, is it true that it might still work, just not validate as xhtml?

**Comment by afred** [comment]
@caseyedavis12 also... are we waiting on legal here?

**Comment by caseyedavis12** [comment]
@afred: Yes, we are waiting on legal. I asked them to make this top priority last week. Will ping.

**Comment by afred** [comment]
waiting on :+1: from legal.

**Comment by mccalluc** [comment]
Legal approved, with a new terms and conditions to be provided by Casey.

File: app/views/catalog/_password_modal.html.erb (hunk 1 above)
Comment (mccalluc) [inline app/views/catalog/_password_modal.html.erb]: yeah, it works fine, it's just the validation that gets confused.

**Comment by mccalluc** [comment]
@afred. This is ready to go. Merge and redeploy?

**Pull request merged**